Editor/CKEditor/CKEditor.php

```diff
@@ -82,16 +82,16 @@ public function getVars()
     private function getDefaults()
     {
         return [
-            'barmode' => 'Standard',
+            'barmode' => 'Extended',
             'height' => '200',
             'resizemode' => 'resize',
             'resizeminheight' => '250',
             'resizemaxheight' => '3000',
             'growminheight' => '200',
             'growmaxheight' => '400',
-            'style_editor' => 'editors/ckeditor/css/contents.css',
+            'style_editor' => '',
             'skin' => 'moono-lisa',
-            'uicolor' => '#D3D3D3',
+            'uicolor' => '',
```

Comment:
Removal on purpose?

Reply:
This colour is also an option. As we moving to new default theme (moono-lisa is now default in CKEditor), then I do not see reason to set special colour. Anyway user can use this field.

Reply:
BTW all other themes generates some small errors in some plugins. It looks like they test mainly with moono-lisa.

Reply:
did you follow the steps outlined in https://github.com/zikula-modules/Scribite/blob/master/Editor/CKEditor/vendor/ckeditor4_upgrade.txt ?

Reply:
My question was related to whether it could be possible that additional plugins do not work properly with the other themes because something could be missing.

Reply:
As per my investigation, some plugins have subdirectory skin, and with some themes generates console errors mainly with missing images. You can see some if change skin.
For example changing to kama:
![image](https://user-images.githubusercontent.com/628801/39229823-e34e8766-486c-11e8-8272-bb75b2946b6d.png)
Anyway the editor works fine.
I noticed some CKEditor integrations in other CMSs moved to moono-lisa.

Reply:
Personally I have no objections against dropping older themes. But this is something to be discussed in Slack I guess.

Reply:
The example above was with moonocolor, not kama, sory.